docs/docs/ref-01-top-level-api.md

```diff
@@ -90,7 +90,11 @@ Verifies the object is a ReactElement.
 
 ### React.Children
 
-`React.Children` provides utilities for dealing with the `this.props.children` opaque data structure.
+`React.Children` provides utilities for dealing with the `this.props.children` data structure.
+
+> **Note:**
+>
+> In most cases, you want to treat `this.props.children` as an opaque object. In reality, its structure can have a number of possibilities. We call it a "React node", which is analogous to DOM nodes which can be elements, text nodes, or fragments containing multiple other nodes. Each React node can be a React element, a string, a number, null/true/false/undefined (all ignored), or an array or iterable containing other React nodes.
```

Comment:
Yes, I agree, but if we're going to be precise and expose it as non-opaque, we might want to break it out into a different section (ie. a section on "The `children` type in JSX").  If we are claiming it's non-opaque, we should describe the behaviors in detail.  What happens if you pass an iterable, for instance?